### PR TITLE
Captain

### PR DESCRIPTION
**Pull request by AmitGithub2006**

### CODE REVIEW

File: src/components/Captain/Captain.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+const renderTooltipContent = (o) => {
+  const { payload = [], label } = o;
+  const total = payload.reduce((result, entry) => result + entry.value, 0);
+
+  return (
+    <div className="customized-tooltip-content">
+      <p className="total">{`${label} (Total: ${total})`}</p>
+      <ul className="list">
+        {payload.map((entry, index) => (
```
Comment (Souvikdeb2612) [inline src/components/Captain/Captain.jsx]: destructure entry here

File: src/components/Captain/Captain.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+};
+
+export function PercentageStackedAreaChart() {
+  const [player, setPlayer] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`https://frozen-harbor-02472.herokuapp.com/captains`)
+      .then((res) => {
+        console.log(res.data);
```
Comment (Souvikdeb2612) [inline src/components/Captain/Captain.jsx]: remove/comment console statements before pushing

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from "react";
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
```
Comment (Souvikdeb2612) [inline src/components/Card/Card.jsx:5]: Format imports

File: src/components/Card/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+}));
+
+export function MainData() {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`https://frozen-harbor-02472.herokuapp.com/series`)
+      .then((res) => {
+        console.log(res.data);
```
Comment (Souvikdeb2612) [inline src/components/Card/Card.jsx:24]: same as above

File: src/components/Card/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        setData([...res.data]);
+      });
+  }, []);
+
+  return (
+    <>
+      <Box sx={{ flexGrow: 1 }} className="Box">
+        <Grid container spacing={2} className="grid-1">
+          <Grid item xs={12} className="grid-2">
+            {data.map((item) => (
```
Comment (Souvikdeb2612) [inline src/components/Card/Card.jsx:34]: destructure item here

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/Captain/Captain.jsx (hunk 1 above)
Comment (AmitGithub2006) [reply src/components/Captain/Captain.jsx]: Issue resolved sir

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**